tests/framework/integration/cluster.go

```diff
@@ -174,6 +174,8 @@ type ClusterConfig struct {
 	ExperimentalMaxLearners     int
 	DisableStrictReconfigCheck  bool
 	CorruptCheckTime            time.Duration
+
+	EnableMemberLogObserver bool
```

Comment:
Don't see any reson this cannot be enabled by default like in e2e package.

Reply:
Updated!